cmake/common/px4_base.cmake

```diff
@@ -743,20 +753,21 @@ function(px4_add_common_flags)
 		${c_compile_flags}
 		${warnings}
 		${c_warnings}
-		${max_optimization}
-		${optimization_flags}
 		${visibility_flags}
 		)
 
 	set(added_cxx_flags
 		${cxx_compile_flags}
 		${warnings}
 		${cxx_warnings}
-		${max_optimization}
-		${optimization_flags}
 		${visibility_flags}
 		)
 
+	set(added_optimization_flags
```

Comment:
Hi bkueng. The added_ variables are used by reconstructing them (as ${added_${lower_var}}), a bit lower (line 1106). I overlooked this at first too! ;)